packages/images-plugin/lib/plugin.js

```diff
@@ -83,6 +82,42 @@ export default class ImageElementPlugin extends BasePlugin {
     // otherwise just do the normal thing
     return await next();
   }
+
+  async performActions(next, driver, ...args) {
+    // Replace with coordinates when ActionSequence includes image elements.
+    const [actionSequences] = /** @type {[import('@appium/types').ActionSequence[]]} */ (args);
```

Comment:
just a small comment to clear out this typescript behaviour.
there is a subtle difference between these two syntaxes:

```js
/** @type {number} */
const x = y;
```

and 

```js
const x = /** @type {number} */ (y);
```

the first one assumes the type of `x` is number and tells the TS compiler to verify if `y` matches this type or raise an error if it does not.

The second one tells TS that `y` must be a number. If `x` was not given a type previously or it is any/unknown then the expression will be compiled successful.

Conclusion: it is always preferable to use the first syntax where possible as it helps to avoid more potential issues. The second instrument is powerful, but sometimes might be breaking if wrong assumptions about types are made, so use with care.